README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Personal Profile
+## Name: Xumeng Yan
+Aspect | Information
+------------ | -------------
+**Pronoun** | She/Her/Hers
+**Program** | MAPSS
+**Major** | Sociology
+**Research Interest** | *Social Stratification, Social Movements*
+
+* A couple things I enjoy:
+  * Experimenting with cooking recipes
+  * Chasing my pets
+  * Lying on the floor and silently listening to music
+
+* Favorite lyrics:
+> *If you want to make the world a better place*
+> *Take a look at yourself, and then make a change* (Michael Jackson)
```

Comment:
Nice use of quote and italics! This quote renders as one line, although your code suggests that it was supposed to be two. In order to start a new paragraph, make sure to end the line with two spaces.